src/controllers/user/home.js

```diff
@@ -24,7 +24,8 @@ define(["homescreenSettings", "userSettingsBuilder", "dom", "globalize", "loadin
                     element: view.querySelector(".homeScreenSettingsContainer"),
                     userSettings: userSettings,
                     enableSaveButton: false,
-                    enableSaveConfirmation: false
+                    enableSaveConfirmation: false,
+                    autoFocus: autoFocuser.isEnabled()
```

Comment:
why do we need it?

Reply:
https://github.com/jellyfin/jellyfin-web/blob/1d0e56c0ae5f74d38ea336cd10ffb9a276069f48/src/components/homescreensettings/homescreensettings.js#L525-L527